core/build.rs

```diff
@@ -0,0 +1,3 @@
+fn main() {
```

Comment:
All the `build.rs` files are missing copyright headers.

Reply:
I will add those.